client/components/editor/Sidebar/index.vue

```diff
@@ -91,6 +87,14 @@ export default {
     metadata() {
       const { repository, selectedElement } = this;
       return getElementMetadata(get(repository, 'schema'), selectedElement);
+    },
+    toolbarClasses() {
```

Comment:
Returning object like before seems simpler 🤔 ?

Reply:
I prefer keeping the template simple (not much logic in it) so that is why I moved the logic to the script part. But I will move it back to the template.